app/views/group_offers/index.html.slim

```diff
@@ -39,6 +41,17 @@ h1= t_title
                     = t_attr(target)
                     br
             td= t("offer_state.#{offer.offer_state}") if offer.offer_state?
+            td
+              - if offer.responsible
+                - if offer.volunteer_responsible_state?
+                  = t("volunteer_responsible_state.#{offer.volunteer_responsible_state}")
+                  br
+                = link_to offer.responsible, volunteer_path(offer.responsible)
+            td
+              - if offer.volunteers.any?
+                - offer.volunteers.each do |volunteer|
+                  = link_to volunteer, volunteer_path(volunteer)
+                  br
```

Comment:
The `if offer.volunteers.any?` isn't really necessary here and might even trigger an additional `COUNT(*)` query, you can also just have the `.each` block and if there are no volunteers it won't output anything.